iogt/static/js/iogt.js

```diff
@@ -0,0 +1,13 @@
+function validateFileUpload(fileInput) {
+  if (!fileInput.files || !fileInput.files[0])
+    return true;
+  else {
+    var file = fileInput.files[0];
+    if (file.size >= 9437184)
```

Comment:
I think this size and the user facing size mentioned in the string below need to be configurable from the backend. @istride can confirm.

Reply:
Yes, that could be useful. @sheralim012 you could start on the Site Settings issue #32 and add this as an option there. But with default 9MB still. It may be best to do this in a separate PR, so that this can be merged as-is.

Reply:
even if we want to use some Site Settings in near future I will strongly suggest to already implement this as const in iogt/iogt/settings/base.py and mark this place with # TODO comment

Reply:
At the moment there is a magic number in the code that needs some explanation. Perhaps assigning that magic number to a constant would help.

Making the warning limit configurable from the same place that other things are configured would make it obvious where  the limit can be changed. I had quite a bit of difficulty getting the warning message to show up in the first place - at first it only worked in incognito/private browsing, but eventually worked outside of that mode, though I'm not sure exactly what I did to make it work. If the warning limit was changed someday, would it require a similar effort from CMS admins to clear their browser cache, in order for the updated limit to take effect?

@sheralim012

Reply:
@istride as Danny mentioned, the file size threshold will be addressed in #32 and I have already created a PR for that and added the threshold there and once this gets merged I'll use that setting here.

If the server is already running and we add/change JS code in templates, sometimes the server doesn't reflect the latest changes and we have to restart the server, and sometimes the browser doesn't reflect the latest changes and we have to clear the browser cache.

Reply:
I don't expect this limit to be changed frequently, if at all, as most sites would use the default. So having to clear the browser cache when changing shouldn't be much of an issue. We should document this though for CMS Editors.